app/controllers/api/v1/groups_controller.rb

```diff
@@ -5,14 +5,32 @@ class Api::V1::GroupsController < Api::V1::BaseApiController
 
   # endpoint to obtain all groups
   def index
-    groups = @assessment.groups
+    show_members = params[:with_members]&.to_s == 'true'
```

Comment:
In the PR description and docs changes you say the param is `show_members` but this is checking for `with_members`. I think we should use `show_members` everywhere.

Also, I think we would prefer something like `ActiveModel::Type::Boolean.new.cast(params[:show_members].downcase)` for the truthy check

Reply:
> In the PR description and docs changes you say the param is `show_members` but this is checking for `with_members`. I think we should use `show_members` everywhere.

Thanks for catching this. I will fix it.

> Also, I think we would prefer something like `ActiveModel::Type::Boolean.new.cast(params[:show_members].downcase)` for the truthy check

The boolean string comparison is used somewhere else in the code, I will update both usages to what you suggested (and others if I find any), unless you prefer I leave that to a different PR.

Reply:
I think you can leave changing the other instances to another PR